W-18068528 add conversation support enhancements mk (#445)
mulesoft/docs-code-builder: modules/ROOT/pages/troubleshoot-generative-ai.adoc

= Troubleshoot Agentforce for Anypoint Code Builder

The following errors sometimes occur when you use Agentforce for Anypoint Code Builder.

== Unexpected Query Results When Using the Agentforce Generative Flow Feature

> If the generative flow is not returning accurate and useful results for the messages you enter, rewrite your message and follow the guidelines to xref:int-ai-create-integrations.adoc#craft-ai-prompt[craft an effective message].

Reviewer: ```suggestion
If the generative flow doesn't return accurate and useful results for the messages you enter, rewrite your message and follow the guidelines to xref:int-ai-create-integrations.adoc#craft-ai-prompt[craft an effective message].
```

== Troubleshoot Agentforce Access Errors

> If you can't access Agentforce, contact your Anypoint Platform organization administrator. If you are the organization administrator, check the following:

Reviewer: ```suggestion
If you can't access Agentforce, contact your Anypoint Platform organization administrator. If you're the organization administrator, check these considerations:
```
